src/app/about/about.component.html

```diff
@@ -0,0 +1,15 @@
+<app-header-static></app-header-static>
+
+<div class="about-content container-top about-container">
+	<img src="../assets/images/about_image.png" />
+	<div class="about-title">About Us</div>
+	<div class="about-content-column col-lg-6 col-md-6 col-sm-12 col-xs-12">
```

Comment:
Row issue @live-wire . Please wrap `col` with in `row`

Reply:
Done!